packages/shared/src/types/props/banner.ts

```diff
@@ -53,6 +53,25 @@ export interface BannerProps extends EmotionJSX.IntrinsicAttributes {
     separateArticleCount?: boolean;
     prices?: Prices;
     choiceCardAmounts?: SelectedAmountsVariant;
+    design?: ConfigurableDesign;
+}
+
+export const configurableDesignSchema = z.object({
+    image: z.object({
+        mobileUrl: z.string(),
+        tabletDesktopUrl: z.string(),
+        wideUrl: z.string(),
+        altText: z.string(),
+    }),
+});
+
+export interface ConfigurableDesign {
+    image: {
```

Comment:
we've actually got an `Image` type already - could this be used?
https://github.com/guardian/support-dotcom-components/blob/main/packages/shared/src/types/props/shared.ts#L137